### DISCUSSION TITLE
Drayton Wiser by Schneider Electric TRV

### DISCUSSION THREAD

**Discussion by fr4ncis3** [Koenkk/zigbee2mqtt#12304] [Devices]
I'm testing a couple of TRVs, Drayton Wiser by Schneider Electric TRV
Model: WV704R0A0902

https://www.zigbee2mqtt.io/devices/WV704R0A0902.html

Firmware version:                    f60f643

Zigbee2MQTT version		1.24.0
commit:                                   7a2ddf24
Coordinator type		       zStack12
	
	

No problem with pairing. But the state of the device reports different values 
from the exposed values described on the page above.
Overall "running_state" and "system_mode" are missing.

Reported state:

`{
    "ADC": "1389,1392,2,215,-32768,0,0,0,100,221,220,225",
    "ALG": "21e,1,40,215,-175,0,0,70,6024,0,100",
    "battery": 100,
    "boost": "Down",
    "last_seen": "2022-04-28T12:13:41+02:00",
    "linkquality": 42,
    "local_temperature": 21.5,
    "occupied_heating_setpoint": 0,
    "pi_heating_demand": 0,
    "voltage": 3.1
}`

Checked previous discussions on the device:

1. #11058
2. #9667
3. #6864

Where  @jamesonuk and @sjorge reports the same issue about missing "system_mode".
So far, no issue about  resetting target temperature raised in discussion #6864.

I  was expecting to use "system_mode" to control the valve, and let my hub to do the logic (not fully 
integrated yet).
By the way, I can control the TRV by setting the "occupied_heating_setpoint"  with a temperature
value higher than the local temperature value, so the TRV fully opens, 
and I can fully close the TRV in a similar way.

Not sure if I'm happy with this device :-)

Regards,
fr4ncis3

**Comment by rob1981forest** [comment]
Hi, I got a few of these and originally had they on the wiser app and I tegrated that way but due to distance I need to pair them to my main ZigBee network. They connect to Z2M ok but report an unknown status in HA and always seem to revert the temperature to 20 degrees at a periodic interval. All I can think to get round this is to make sure I keep pushing the temperature I want to set it at every 5 minutes to make sure this glitch doesn't affect it.......as it's getting warmer now I probably won't be able to test with the heating on before the winter now !!! 

If we could get this issue where the desired temp reverts to 20 periodically I think these would work good.

**Comment by fr4ncis3** [comment]
Unfortunately, in my experience:

*It is NOT possible to simply  OPEN / CLOSE the valve.
You need to play with the temperature setpoint to open or close the TRV.

* Floods the ZigBee network with too many messages.
Tried to reduce chattiness e set debounce but no luck

I think this TRV is designed to work flawlessly with his own hub.

I've sent my items back


regards,
fr4ncis3
